homeassistant/components/tuya/cover.py

```diff
@@ -229,21 +230,34 @@ def __init__(
                 device.status_range[tilt_dpcode].values
             )
 
+        # Determine current_position DPCodes
+        if self.entity_description.current_position is None:
+            if self.entity_description.set_position is not None:
+                self._position_dpcode = self.entity_description.set_position
+            else:
+                self._position_dpcode = None
```

Comment:
These can be combined, since `None` is the default for `_position_dpcode` already
```suggestion
        if self.entity_description.current_position is None and self.entity_description.set_position is not None:
            self._position_dpcode = self.entity_description.set_position
```